README.md

```diff
@@ -7,34 +7,46 @@ The Backed community gives out NFTs to recognize community members and contribut
 
 The [first NFT type](https://opensea.io/assets/0xb9a069f85d51cc5cf23e910d1a5fd69744e926f1/1) was given out to those who used the product on Rinkeby in fall 2021. 
 
-In February 2022, we decided on some broad categories of community NFTs going forward, and various levels for each category. The current types of NFTs are 
+In February 2022, we decided on some broad categories of community NFTs going forward, and various levels for each category. Those types of NFTs were: 
 - **Trusted Tester**, given to those who use the product and catch a bug or provide some helpful feedback.
 - **Pawn Shopper**, given to those who are active in discord or community calls.
 - **Design Community**, given to those who contribute to design conversations and contribute design assets.
 - **Code Contributor**, given to those who contribute code that is deployed to production.
 
-New types will be added in the future as there is need. It is expected that each contributor will at most get a unique token/level once, e.g. you cannot get multiple Level 1 Trusted Tester tokens.
+In June 2022, we migrated to a new Community NFT that tracks XP across three categories: Activity, Contributor, and Community, and awards them both automatically (via bots) and through a nomination process. This rubric outlines eligibility for XP rewards: 
```

Comment:
I wonder if we need to keep tracking like this, or if people should just check the commit history? Or we could preserve old READMEs in a file somewhere?